src/OrchardCore.Modules/OrchardCore.Workflows/Trimming/Services/WorkflowTrimmingManager.cs

```diff
@@ -5,29 +5,29 @@
 using OrchardCore.Settings;
 using OrchardCore.Workflows.Indexes;
 using OrchardCore.Workflows.Models;
-using OrchardCore.Workflows.WorkflowPruning.Models;
+using OrchardCore.Workflows.Trimming.Models;
 using YesSql;
 using YesSql.Services;
 
-namespace OrchardCore.Workflows.WorkflowPruning.Services;
+namespace OrchardCore.Workflows.Trimming.Services;
 
-public class WorkflowPruningManager : IWorkflowPruningManager
+public class WorkflowTrimmingManager : IWorkflowTrimmingManager
 {
     private readonly ISiteService _siteService;
     private readonly ISession _session;
     private readonly IClock _clock;
 
-    public WorkflowPruningManager(ISiteService siteService, ISession session, IClock clock)
+    public WorkflowTrimmingManager(ISiteService siteService, ISession session, IClock clock)
```

Comment:
I would span these properties across multiple lines. One service per line

Reply:
It's a short line, why would we do that?

Reply:
For consistency with what we have been doing everywhere else